### PR TITLE
ogg,vorbis: Install library files

### PR DESCRIPTION
**Pull request by seungha-yang**
Add missing install statement

### CODE REVIEW

**Comment by tp-m** [comment]
You also need to update `releases.json` to add new revisions for each wrap.

And while you're at it, please fix the version number in vorbis's `meson.build` `project()`, should be 1.3.7 (as per https://github.com/mesonbuild/wrapdb/pull/201#issuecomment-945027960)

**Comment by tp-m** [comment]
Also, what about the header files and pkg-config files?

I don't see any `install_headers()` calls etc.

File: subprojects/packagefiles/ogg/src/meson.build [hunk 1]
```diff
@@ -4,6 +4,7 @@ libogg = library('ogg',
   sources,
   include_directories : incdir,
   vs_module_defs: '../win32/ogg.def',
+  install: true,
```
Comment (eli-schwartz) [inline subprojects/packagefiles/ogg/src/meson.build:7]: While we are at it, the official build system installs this with a soname "0" and a full version 0.8.5

File: subprojects/packagefiles/vorbis/lib/meson.build [hunk 2]
```diff
@@ -31,37 +31,59 @@ if host_machine.system() == 'windows'
 endif
 
 math_dep = meson.get_compiler('c').find_library('m', required: false)
+pkgconfig = import('pkgconfig')
 
 vorbis_lib = library('vorbis', vorbis_sources,
   include_directories : [incdir],
   dependencies : [ogg_dep, math_dep],
   vs_module_defs: '../win32/vorbis.def',
+  install: true,
```
Comment (eli-schwartz) [inline subprojects/packagefiles/vorbis/lib/meson.build:40]: Upstream installs as libvorbis.so.0.4.9

File: subprojects/packagefiles/vorbis/lib/meson.build [hunk 3]
```diff
@@ -31,37 +31,59 @@ if host_machine.system() == 'windows'
...
+  description: 'vorbis is the primary Ogg Vorbis library',
+  version: meson.project_version()
+)
+
 vorbisfile_lib = library('vorbisfile', vorbisfile_sources,
   include_directories : [incdir],
   link_with: vorbis_lib,
   dependencies : [ogg_dep, math_dep],
   vs_module_defs: '../win32/vorbisfile.def',
+  install: true,
```
Comment (eli-schwartz) [inline subprojects/packagefiles/vorbis/lib/meson.build:58]: libvorbisfile.so.3.3.8

File: subprojects/packagefiles/vorbis/lib/meson.build [hunk 4]
```diff
@@ -31,37 +31,59 @@ if host_machine.system() == 'windows'
...
+  description: 'vorbisfile is a library that provides a convenient high-level API for decoding and basic manipulation of all Vorbis I audio streams',
+  version: meson.project_version()
+)
+
 vorbisenc_lib = library('vorbisenc', vorbisenc_sources,
   include_directories : [incdir],
   link_with: vorbis_lib,
   dependencies : [ogg_dep, math_dep],
   vs_module_defs: '../win32/vorbisenc.def',
+  install: true,
```
Comment (eli-schwartz) [inline subprojects/packagefiles/vorbis/lib/meson.build:75]: libvorbisenc.so.2.0.12

**Comment by seungha-yang** [comment]
rebased

**Pull request merged**